simvue/run.py

```diff
@@ -262,6 +258,13 @@ def init(self, name=None, metadata={}, tags=[], description=None, folder='/', ru
 
         self._check_token()
 
+        # compare with pydantic RunInput model    
+        import json 
+        try:
+            runinput = RunInput(**data)
+        except ValidationError as e:
+            self._error(json.dumps(e.json()))        
```

Comment:
I think using instead here:
```
            self._error(e)
```
rather than:
```
            self._error(json.dumps(e.json()))
```
makes things much easier to understand. Using json you get an error like this:
```
RuntimeError: "[\n  {\n    \"loc\": [\n      \"metadata\",\n      \"hello\"\n    ],\n    \"msg\": \"str type expected\",\n    \"type\": \"type_error.str\"\n  },\n  {\n    \"loc\": [\n      \"metadata\",\n      \"hello\"\n    ],\n    \"msg\": \"value is not a valid integer\",\n    \"type\": \"type_error.integer\"\n  },\n  {\n    \"loc\": [\n      \"metadata\",\n      \"hello\"\n    ],\n    \"msg\": \"value is not a valid float\",\n    \"type\": \"type_error.float\"\n  }\n]"
```
whereas with plain text you get:
```
RuntimeError: 3 validation errors for RunInput
metadata -> hello
  str type expected (type=type_error.str)
metadata -> hello
  value is not a valid integer (type=type_error.integer)
metadata -> hello
  value is not a valid float (type=type_error.float)
```
For me at least this is easier to understand.

Reply:
Yes, agreed. Will update.